stable/dask/templates/dask-jupyter-config.yaml

```diff
@@ -0,0 +1,18 @@
+{{ if .Values.jupyter.enabled -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "dask.fullname" . }}-jupyter-config
+  labels:
+    app: {{ template "dask.name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: {{ template "dask.chart" . }}
+    component: "{{ .Release.Name }}-{{ .Values.jupyter.component }}"
```

Comment:
Let me ask again why you make components configurable. Also, they should not be tied to a release. I'd just hard-code them: `jupyter`, `worker`

Reply:
Thanks for the nudge.  Resolved.  My apologies for letting that slip earlier. 

To answer the direct question of why was this here.  My answer is "I don't know".  This PR is a copy-paste-modify job of the current chart in `stable/dask-distributed`, that can hopefully be deprecated once this cleaner version is in.